src/formatDate.js

```diff
@@ -8,7 +8,55 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newFormat = [];
+  const separatorFrom = fromFormat[3];
+  const separatorTo = toFormat[3];
+  const numbers = date.split(separatorFrom);
+  let [year, month, day] = numbers;
```

Comment:
The initial assignment of `year`, `month`, and `day` assumes a specific order in the `numbers` array. This may not be correct if the `fromFormat` is different. Ensure that the assignment matches the `fromFormat` specification.